GameServerLib/Content/CharData.cs

```diff
@@ -10,8 +13,32 @@ namespace LeagueSandbox.GameServer.Content
     public class PassiveData : IPassiveData
     {
         public string PassiveNameStr { get; set; } = "";
-        public string PassiveLuaName { get; set; } = "";
+        public string PassiveAbilityName { get; set; } = "";
         public int[] PassiveLevels { get; set; } = { -1, -1, -1, -1, -1, -1 };
+
+        //TODO: Extend into handling several passives, when we decide on a format for that case.
+        public static string GetPassiveAbilityNameFromScriptFile(string champName, List<IPackage> packages)
+        {
+            foreach (var package in packages)
+            {
+                var path = $"{package.PackagePath}\\Champions\\{champName}\\Passive.cs";
+
+                if (File.Exists(path))
+                {
+                    var inputPassiveFile = File.ReadAllText(path);
+
+                    string pattern = @"class (?<passiveName>\w+) : IGameScript";
```

Comment:
We need to discuss whether we want to use a way like that (inferring names from classes. Maybe we want to do some reflection magic to have a better handling)

Reply:
I don't have much experience using the Reflections library, could you explain how you would want to use it for this use-case`?

Reply:
See this https://github.com/LeagueSandbox/GameServer/blob/indev/GameServerLib/Scripting/CSharp/CSharpScriptEngine.cs (this class use a lot of reflection).

Reply:
Let's delay it for another PR as compiling in real time only to find the passive name is not efficient at all. (We need to add functions to the `CSharpScriptEngine` to allow that)